### PR TITLE
Update Performance Points

### PR DESCRIPTION
**Pull request by MilkyIQ**
Updates the performance points article to mention the recent update to the pp system in the history section. also applied some courtesy spellchecking on the rest of the article

### CODE REVIEW

File: wiki/Performance_points/en.md [hunk 1]
```diff
@@ -19,9 +14,18 @@ Known thereafter as "pp" (an abbreviation for "performance points"), this new sy
...
 
-It continued to exist in this capacity for more than a year of service, until [Tom94](https://osu.ppy.sh/users/1857058), the creator of the *osu!tp* scoring metric, joined the [osu! team](/wiki/People/The_Team) and implemented his design into the system. The resulting system was titled *ppv2*, and became live on the January 27, 2014. Therfore renaming the old system to *[ppv1](/wiki/Performance_points/ppv1)*
+On January 16, 2021, changes were made to the ppv2 system that aimed to more accurately award pp to more difficult aspects of maps. These changes were made in large part by the help of various individual members of the community such as [Xexxar](https://osu.ppy.sh/users/2773526) and [StanR](https://osu.ppy.sh/users/7217455). The specifics of the changes made are detailed in its [corresponding newspost](https://osu.ppy.sh/home/news/2021-01-14-performance-points-updates). Very briefly, the main points of interest in the update were as follows:
```
Comment (TicClick) [inline wiki/Performance_points/en.md:21]:
```suggestion
On January 16, 2021, changes were made to the ppv2 system that aimed to more accurately award pp to more difficult aspects of maps. These changes were made in large part by the help of various individual members of the community such as [Xexxar](https://osu.ppy.sh/users/2773526) and [StanR](https://osu.ppy.sh/users/7217455). The specifics of the changes made are detailed in the[corresponding newspost](https://osu.ppy.sh/home/news/2021-01-14-performance-points-updates). Very briefly, the main points of interest in the update were as follows:
```

File: wiki/Performance_points/en.md [hunk 2]
```diff
@@ -19,9 +14,18 @@ Known thereafter as "pp" (an abbreviation for "performance points"), this new sy
...
+
+- Increase the amount of pp gained in higher approach rate maps
```
Comment (TicClick) [inline wiki/Performance_points/en.md:23]: it's not always an increase:

> *The buff for AR 11 maps now scales from 0% at 0 objects, to 26.66% at a maximum of 1000 objects. The previous 20% buff is now attained at around 750 objects instead.*

meaning that in most cases pp are reduced

File: wiki/Performance_points/en.md [hunk 3]
```diff
@@ -19,9 +14,18 @@ Known thereafter as "pp" (an abbreviation for "performance points"), this new sy
...
+- Adjust the pp loss from use of the [No Fail](/wiki/Game_modifier/No_Fail) mod
+- Adjust the pp loss from use of the [Spun Out](/wiki/Game_modifier/Spun_Out) mod
+- Adjust the rate of pp loss from misses to be more forgiving on longer maps with higher combo
```
Comment (TicClick) [inline wiki/Performance_points/en.md:26]: judging by the graph in the news article, it's actually less forgiving

File: wiki/Performance_points/en.md [hunk 4]
```diff
@@ -19,9 +14,18 @@ Known thereafter as "pp" (an abbreviation for "performance points"), this new sy
...
+- Increase the amount of pp gained from player [accuracy](/wiki/Accuracy)
```
Comment (TicClick) [inline wiki/Performance_points/en.md:27]: not really, it's more about higher pp loss on bad accuracy

File: wiki/Performance_points/en.md [hunk 5]
```diff
@@ -19,9 +14,18 @@ Known thereafter as "pp" (an abbreviation for "performance points"), this new sy
...
+- Decrease the amount of pp gained from maps with higher speed
```
Comment (TicClick) [inline wiki/Performance_points/en.md:28]: uhh, where do you see that? all I can find is higher penalty from excessive amount of 50s

**Review by TicClick — CHANGES_REQUESTED** [review]

File: wiki/Performance_points/en.md (hunk 5 above)
Comment (MilkyIQ) [reply wiki/Performance_points/en.md:28]: this is badly worded and should be removed.

(i was trying to mention the update's reduced focus on high speed)

File: wiki/Performance_points/en.md (hunk 3 above)
Comment (MilkyIQ) [reply wiki/Performance_points/en.md:26]: i fail to see how thats true tbh

> This change seeks to penalize misses on short maps more, while significantly reducing the impact of a small number of misses on longer maps.

Comment (TicClick) [reply wiki/Performance_points/en.md:26]: sigh. you're right, I assumed that was a **score** multiplier plot, such as on the previous graphs

**Review by TicClick — APPROVED** [review]

**Pull request merged**